src/flinkStatementResultsManager.ts

```diff
@@ -81,6 +83,7 @@ export type PostFunction = {
     isForeground: boolean;
   }>;
   (type: "StopStatement", body: { timestamp?: number }): Promise<null>;
+  (type: "ViewStatementSource", body: { timestamp?: number }): Promise<null>;
```

Comment:
Am ... unusure if we even need a body? StopStatement doesn't use its body timestamp either ;-(.

Reply:
Rohit says no, was carried over from message viewer implementation, who may well also not need it in most places. New ticket https://github.com/confluentinc/vscode/issues/1912 written.